src/current/cockroachcloud/custom-metrics-chart-page.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Custom Metrics Chart Page
+summary: The Custom Metrics Chart page lets you create custom charts showing the time series data for an available metric or combination of metrics.
+toc: true
+---
+
+The **Custom Metrics Chart** page is available only for CockroachDB {{ site.data.products.serverless }} clusters. To view this page, select a cluster from the [**Clusters** page]({% link cockroachcloud/cluster-management.md %}#view-clusters-page), and click **Metrics** in the **Monitoring** section of the left side navigation. Navigate to the **Custom** tab to create custom charts showing the time series data for an available metric or combination of metrics.
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+## Use the Custom Metrics Chart page
+
+<img src="{{ 'images/cockroachcloud/custom-metrics-chart.png' | relative_url }}" alt="Custom Metrics Chart" style="border:1px solid #eee;max-width:100%" />
+
+On the **Custom Metrics Chart** page, you can set the time range for all charts, add new custom charts, and customize each chart:
+
+- To set the time range for the page, use the time interval selector at the top right of the page to filter the view for a predefined or custom time interval. Use the navigation buttons to move to the previous, next, or current time interval. When you select a time interval, the same interval is selected for all charts on all tabs of the [**Metrics** page]({% link cockroachcloud/metrics-page.md %}).
+- To add a chart, click **Create** to create the first custom chart or **Add Chart** to create subsequent custom charts. The [**Create custom chart** dialog](#create-custom-chart-dialog) is displayed.
+- To edit a chart, click the pencil icon to display the [**Edit custom chart** dialog](#create-custom-chart-dialog).
+- To delete a chart, click the trash icon.
+
+## Create custom chart dialog
+
+In the **Create custom chart** dialog, you can customize each chart.
+
+<img src="{{ 'images/cockroachcloud/custom-metrics-chart-create.png' | relative_url }}" alt="Create Custom Chart" style="border:1px solid #eee;max-width:70%" />
+
+- Under **Select metrics**, add the metrics to be queried, and how they'll be combined and displayed. Options include:
```

Comment:
Not sure if here or in the "Units" section within the column/description table makes more sense, but it would be useful to mention that available metric options are filtered by unit. 

Specifically:

Unit selection can be populated 2 ways: 
- by selecting a unit option
- when the first metric has been added - if no unit has been selected yet the metric's unit will populate the unit selection

Once a unit is selected, the available metric options will be filtered to metrics of that unit type (i.e. if the `BYTES` unit is selected, only metrics that are measured in `BYTES` will be available as metric options)

Note that changing the unit will clear the selected metrics.

Reply:
Just remembered this - might be good to mention the `Clear metrics` button as well (which effectively resets the modal to a blank state - no metrics, unit, or aggregators set)

Reply:
Updated descriptions for both the Metric and Units rows of the columns/description table.